integration_tests/jacoco-offline/src/test/java/org/robolectric/integrationtests/mockk/Shadow.java

```diff
@@ -0,0 +1,15 @@
+package org.robolectric.integrationtests.mockk;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+import org.robolectric.integrationtests.jacoco.UnderShadow;
+
+@Implements(UnderShadow.class)
+public class Shadow {
```

Comment:
I don't recommend you to use the class name `Shadow`. What about using name like `JaCoCoOfflineTester` and `ShadowJaCoCoOfflineTester` or other proper names?

Reply:
Good idea! The name is good.